src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -60,22 +60,37 @@ public void SortedSet_Generic_Constructor_IEnumerable_Null()
         public void SortedSet_Generic_Constructor_IEnumerable_IComparer(EnumerableType enumerableType, int setLength, int enumerableLength, int numberOfMatchingElements, int numberOfDuplicateElements)
         {
             IEnumerable<T> enumerable = CreateEnumerable(enumerableType, null, enumerableLength, 0, 0);
+#if netcoreapp
```

Comment:
>netcoreap [](start = 4, length = 9)

could you please move the netcoreapp specific tests to its own file instead of using #if's?